day-26-challenge/__test__/solution.test.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const fp = require('../solution.js');
+
+require('jest');
+
+let myArray = [1, 2, 3, 4, 5];
+let myArray2 = [1, 2, 3, 4, 5];
+let myArray3 = [1, 2, 3, 4, 5];
+let notArray = { 1: 'one', 2: 'two', 3: 'three', 4: 'four', 5: 'five' };
+let callback = function (a, b) {
```

Comment:
How about more specific naming practices. What does this callback do? It sums things, or add things, so call it add or adder or something like that.